tasks.py

```diff
@@ -33,7 +33,7 @@ def run_api(c):
 
 @task
 def run_worker(c):
-    c.run("python -m erica.infrastructure.rq.worker")
+    c.run("huey_consumer.py erica.infrastructure.huey.huey.huey -k thread -w 10")
```

Comment:
We don't want the number of workers to be configurable? (Or maybe `-w` means a completely different thing. :D )